.github/workflows/kentik-api.yml

```diff
@@ -78,6 +78,16 @@ jobs:
         run: |
           pip3 install -r requirements.txt
           python3 setup.py build sdist bdist_wheel
+      - name: Run Super-Linter
+        uses: github/super-linter@v3.17.0
+        env:
+          LINTER_RULES_PATH: /.github/linters
+          DEFAULT_BRANCH: master
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          VALIDATE_JSCPD: false
+          VALIDATE_PYTHON_ISORT: false
```

Comment:
I sort is already configured and running on CI. We shouldn't disable it if we use super-linter for other python linters

Reply:
Done